### DISCUSSION TITLE
Sales and discounts

### DISCUSSION THREAD

**Discussion by simonj** [etsy/open-api#1153] [Q&A]
Are there any API calls I can use to create Sales and discounts?

**Comment by simonj** [comment]
Any etsy developers here 🙂?

**Comment by Gareth064** [comment]
There are no Developers here that work for Etsy :) Just a bunch of people trying to use their API and helping each other when we can.

As for your question, if you don't see an endpoint [here](https://developers.etsy.com/documentation/reference), then there is none. (I just checked and can confirm there is none)

**Reply by simonj** [reply]
Dang. Thanks a lot @Gareth064
